Sources/TSCBasic/Process/Process.swift

```diff
@@ -323,21 +376,34 @@ public final class Process {
     ///     continue running even if the parent is killed or interrupted. Default value is true.
     ///   - loggingHandler: Handler for logging messages
     ///
+    public init(arguments: [String], environmentBlock: ProcessEnvironmentBlock = ProcessEnv.block, workingDirectory: AbsolutePath, outputRedirection: OutputRedirection = .collect, startNewProcessGroup: Bool = true, loggingHandler: LoggingHandler? = .none) {
```

Comment:
formatting nit for long lines
```suggestion
    public init(
        arguments: [String], 
        environmentBlock: ProcessEnvironmentBlock = ProcessEnv.block, 
        workingDirectory: AbsolutePath, 
        outputRedirection: OutputRedirection = .collect, 
        startNewProcessGroup: Bool = true, 
        loggingHandler: LoggingHandler? = .none
    ) {
```